app-emulation/firecracker-bin/Manifest

```diff
@@ -2,3 +2,7 @@ DIST firecracker-v1.4.0-aarch64.tgz 8415004 BLAKE2B 8311a8ec4a98be63ca6398559bab
 DIST firecracker-v1.4.0-x86_64.tgz 8818170 BLAKE2B 0f5ed139ad6426f45e31b1c29238199a67af2140a8c8992779472563cd4dc75d0311161f0bf95dd708a9660b2f3d55e5f89f4c0e8e5767958f98cf57a4b2157f SHA512 e72aafcc667a78ac691000e81d67aaf0cc69adbbefaed19e647d294988a084612b0c3b6f73738200e8b5769cc128b07e0bf8bc1a079402f11317c70e17147594
 DIST firecracker-v1.4.1-aarch64.tgz 8517732 BLAKE2B 4c12716654a48aaf82c1b8888ad26449f2d9f8ffb9d002707dc921de9ec72a25b6afb5dd6f3d65cd7857de3ca5f65e9eede9f352a4aab55d4fe7cccbf939c201 SHA512 c8b83515d8e84c6bb42c5e9a7a8f08bd7eb5a91d7e255551849ff703f1867ab30e7e69dc139215f138035e253e853367697ff431e67dc868ec5cacce2b5762b2
 DIST firecracker-v1.4.1-x86_64.tgz 8957780 BLAKE2B ba6f9a4a01157e3e5e93a3e8676ec7b32cd40a9f9dac101c441e359e965db02adaa2f1c439f98732ac66c4b65b384db0d1fa8032d605a2db7377ab3691a09a06 SHA512 a56db43bf725c547ad5fae15c72ebb669fc66e037f63bc30048a1287d4f0b49a2841949693da23af0359ba2d2e43e82ef8c016a88f3fa2ced311580e3bd98fee
+DIST firecracker-v1.5.0-aarch64.tgz 9237991 BLAKE2B a53860befdf8d14509abeb4b1f85df535791e8c925f84acae332a954376a2abb4c3f0cbda4b48125eb9c980d060e3854877dc54af1d10be0d99160221b71a907 SHA512 aa16242b72e609fd5275fceb26fc83db874131eff883bfb3d3c4a5396d269c1e34a8d69bdd61b102be5ca30fc50da9dab131e853716c6a64072f7c919fb86d48
+DIST firecracker-v1.5.0-x86_64.tgz 9594647 BLAKE2B 6c4f13fff13b722444150c82ab18838f8c9a60d3b11cfe71ef30d3968ae1a15fefcfd9908c5ce1dc1fe2cf3c449a02cfd89dea6f40bb2eb9a0eb0db050267ac2 SHA512 899685e149aa9a7677fdcd63d923d51fd6a8aa09fc467e9f36d0375f204557eae212a249b2c8af01bf931729aa3cedb871ba2a0a2249813b135738f0bc60a36f
```

Comment:
Not really sure we need 1.5.0 at all, can you drop the commit for it?

Reply:
You're right, we probably don't need it. Removed.